### PR TITLE
ENH: Improvements in the Sample Data module

### PR DESCRIPTION
**Pull request by cpinter**
- Made possible to load multiple files with multiple loaders
- Add Development category that only shows up in Developer mode
- Add one dataset in the Development category: TinyPatient, containing a 10x10x10 CT and a binary labelmap segmentation with two segments. Very useful for testing
- Stop trying to download data after five attempts (it could get into infinite loop)

### CODE REVIEW

File: Modules/Scripted/SampleData/SampleData.py [hunk 1]
```diff
@@ -289,10 +298,22 @@ def registerBuiltInSampleDataSources(self):
...
+      slicer.modules.sampleDataSources[self.builtInCategoryName].append(SampleDataSource(*sourceArgument))
+
+  def registerDevelopmentSampleDataSources(self):
+    iconPath = os.path.join(os.path.dirname(__file__).replace('\\','/'), 'Resources','Icons')
+    self.registerCustomSampleDataSource(
+      category=self.developmentCategoryName, sampleName='TinyPatient',
+      uris=['http://slicer.kitware.com/midas3/download/item/245205/TinyPatient_CT.nrrd',
+            'http://slicer.kitware.com/midas3/download/item/367020/TinyPatient_Structures.seg.nrrd'],
+      fileNames=['TinyPatient_CT.nrrd', 'TinyPatient_Structures.seg.nrrd'],
+      nodeNames=['TinyPatient_CT', 'TinyPatient_Structures'],
```
Comment (pieper) [inline Modules/Scripted/SampleData/SampleData.py]: TinyPatient_Segments?

**Comment by pieper** [comment]
LGTM - 👍

File: Modules/Scripted/SampleData/SampleData.py (hunk 1 above)
Comment (cpinter) [reply Modules/Scripted/SampleData/SampleData.py]: Yes, a body and a tumor. Everything is super low resolution for testing

Comment (cpinter) [reply Modules/Scripted/SampleData/SampleData.py]: Oh I see, the name... OK :)

**Comment by cpinter** [comment]
Thanks a lot for the quick review, Steve!

Integrated in https://github.com/Slicer/Slicer/commit/be752e458c4ae9f53c0716d330f4d1c6cf4b6c92

**Pull request closed without merging**